docs/rules.adoc

```diff
@@ -0,0 +1,46 @@
+ifdef::env-github,env-browser[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+:outfilesuffix: .adoc
+endif::[]
+
+=== Rules
+:icons: font
+
+If you want to use arquillian functionality with some other runner. You can use Arquillian with Junit Rule using `ArquillianTestClass` and `ArquillianTest` Rules.
+
+In order to use full functionality of Arquillian you have to use both the rules i.e. `ArquillianTestClass` and
+`ArquillianTest`.
```

Comment:
What if I only use one? Shouldn't we require having both to have similar functionality to what the `@RunWith(Arquillian.class)` has to offer?

Reply:
ok I will update comment